infra/README.md

```diff
@@ -0,0 +1,139 @@
+
+**~/.aws/credentials** (Linux & Mac) or **%USERPROFILE%\.aws\credentials** (Windows)
+
+### Examples:
+```
+$ aws configure
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-2
+Default output format [None]: json
+```
+**Using the above command will create a [default] profile.**  
+```
+$ aws configure --profile dev
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+Default region name [None]: us-east-2
+Default output format [None]: json
+```
+**Using the above command will create a [dev] profile.**  
+
+### References:
+
+- [Configuration basics][1]
+- [Named profiles for the AWS CLI][2]
+- [Configuration and credential file settings][3]
+
+[1]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html
+[2]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html
+[3]: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html
+
+## Basic Terraform Commands 
+
+&nbsp;&nbsp;The `terraform init` command is used to initialize a working directory containing Terraform configuration files. This is the first command that should be run after writing a new Terraform configuration or cloning an existing one from version control. It is safe to run this command multiple times.
+
+&nbsp;&nbsp;The `terraform plan` command creates an execution plan, which lets you preview the changes that Terraform plans to make to your infrastructure. By default, when Terraform creates a plan it:
+
+- Reads the current state of any already-existing remote objects to make sure that the Terraform state is up-to-date.
+- Compares the current configuration to the prior state and noting any differences.
+- Proposes a set of change actions that should, if applied, make the remote objects match the configuration.
+
+&nbsp;&nbsp;The `terraform apply` command executes the actions proposed in a Terraform plan.
+
+<span style="color:red"> ---- DANGER ----</span>\
+In a non-development environment comment out a resource to destroy, destroy should only be used as a way to cleanup a development environment. e.g. a developers workspace after they are done with it.
+
+&nbsp;&nbsp; The `terraform destroy` command is a convenient way to destroy all remote objects managed by a particular Terraform configuration.\
+
+For more information about terraform commands follow the link below:
+
+- [Basic CLI Features](https://www.terraform.io/cli/commands)
+
+## Terraform Backend
+
+### infra/bootstrap/account
+
+1. Rename bootstrap/account, account directory to the name of the aws account alias or account id where this infrastructure will be hosted.
```

Comment:
Would the intent here for recommended usage be that we have a 1:1 relationship between `bootstrap/account` folders and `envs/environment` folders?

i.e.

- `bootstrap/my-project-test` and `envs/test`
- `boostrap/my-project-staging` and `envs/staging`
- etc.

Just confirming my understanding since the PR has multiple env folders and one bootstrap folder at the moment. thx!

Reply:
The relationship would be 1:1 in a multi-account setup. If a project were required to host multiple environments in a single account, then all environments would end up pointing to a single bootstrap/account backend. These drawings below will be added to this document as a visual aid.

See link below: 
https://lucid.app/lucidchart/66ee71f6-e9e1-41e3-97b6-7bd4c80b6670/edit?viewport_loc=-3561%2C173%2C2736%2C1423%2C0_0&invitationId=inv_e4cd8c17-8242-4cc2-ba4a-92b4ef8d8d44#

Reply:
the diagram is super helpful, thank you!